VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.non-event.spec.ts

```diff
@@ -427,72 +429,110 @@ describe('WaitingRoomComponent message and clock', () => {
         expect(roomClosingToastrService.currentToast).toBeTruthy();
     });
 
-    it('should use interpreter room when participant has links', async () => {
-        component.participant.linked_participants = [
-            new LinkedParticipantResponse({ linked_id: Guid.create().toString(), link_type: LinkType.Interpreter })
-        ];
-        const room = new SharedParticipantRoom({
-            participant_join_uri: 'patjoinuri',
-            pexip_node: 'sip.test.node',
-            display_name: 'foo',
-            tile_display_name: `I1;Interpreter1;${component.participant.id}`
+    describe('call', () => {
+        beforeAll(() => {
+            jasmine.getEnv().allowRespy(true);
+        });
+        afterAll(() => {
+            jasmine.getEnv().allowRespy(false);
         });
-        videoCallService.retrieveInterpreterRoom.and.resolveTo(room);
 
-        await component.call();
+        describe('failing scenarios', () => {
+            beforeEach(() => {
+                videoCallService.makeCall.calls.reset();
+                spyOnProperty(eventsServiceSpy, 'eventHubIsConnected').and.returnValue(false);
+                component.token = undefined;
+            });
 
-        expect(videoCallService.makeCall).toHaveBeenCalledWith(
-            room.pexip_node,
-            room.participant_join_uri,
-            room.tile_display_name,
-            component.maxBandwidth
-        );
-    });
+            it('should not call when event hub is not connected and token is not set', () => {});
```

Comment:
has this been left by mistake? Looks to be an empty test block

Reply:
The setup for this are handled by the beforeEach of the describe. I'll at least add a comment but I could see an argument that setup would be clearer if it was handled individually.